lib/Backend/arm64/LowerMD.cpp

```diff
@@ -3323,7 +3323,7 @@ LowererMD::GenerateFastAdd(IR::Instr * instrAdd)
     }
 
     // s1  = SUB src1, 1     -- get rid of one of the tag
-    opndReg = IR::RegOpnd::New(TyInt32, this->m_func);
```

Comment:
>TyInt32 [](start = 31, length = 7)

We are doing a Int add TyInt32 is ok here. What was the purpose of making it MachReg?